### PR TITLE
Update script to enforce consistency and fix small errors during cleanup

### PR DESCRIPTION
**Pull request by Brunoga-MS**
@jfaurskov : Hello Jan, can you please validate and approve?

### CODE REVIEW

File: docs/wiki/Deploy-with-Azure-PowerShell.md [hunk 1]
```diff
@@ -112,19 +112,20 @@ Run only the commands that correspond to your management group hierarchy.
   New-AzManagementGroupDeployment -ManagementGroupId $pseudoRootManagementGroup -Location $location -TemplateFile ./infra-as-code/bicep/deploy_dine_policies.bicep
   
   #Deploy policy initiatives, wait approximately 1-2 minutes after deploying policies to ensure that there are no errors when creating initiatives
```
Comment (arjenhuitema) [inline docs/wiki/Deploy-with-Azure-PowerShell.md:114]: We intentionally deploy the PolicyDefinitions and the PolicySetDefinitions to the pseudo root management group. Only the assignments are made to the specific management groups.

**Review by arjenhuitema — CHANGES_REQUESTED** [review]
Thank you Bruno, looks good!

Can you provide some evidence on running the cleanup script. What was the error that is solved with this? 

Please remove the changes to docs/wiki/Deploy-with-Azure-PowerShell.md the deployment location is intentional.

File: docs/wiki/Deploy-with-Azure-PowerShell.md (hunk 1 above)
Comment (Brunoga-MS) [reply docs/wiki/Deploy-with-Azure-PowerShell.md:114]: Hey @arjenhuitema,
changes to docs/wiki/Deploy-with-Azure-PowerShell.md the removed. This is the error you get with the force parameter on the Remove-AzRoleAssignment
image

**Review by arjenhuitema — APPROVED** [review]
LGTM

**Pull request merged**